docs/configuration-parameters.md

```diff
@@ -58,7 +58,7 @@ This section provides the details of all the configurations required for CarbonD
 | carbon.merge.sort.prefetch | true | Enable prefetch of data during merge sort while reading data from sort temp files in data loading. |  |
 | carbon.update.persist.enable | true | Enabling this parameter considers persistent data. Enabling this will reduce the execution time of UPDATE operation. |  |
 | carbon.load.global.sort.partitions | 0 | The Number of partitions to use when shuffling data for sort. If user don't configurate or configurate it less than 1, it uses the number of map tasks as reduce tasks. In general, we recommend 2-3 tasks per CPU core in your cluster.
-
+| carbon.enable.vector.reader | true | This parameter increases the query performance. | |
```

Comment:
I think it is better to give more information about this option, perhaps you can describe why it can increase performance and give hint when to enable this feature